shells/electron/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "main": "./index.js",
+  "scripts": {
+    "build": "webpack --config webpack.backend.js && webpack --config webpack.config.js"
```

Comment:
what does this expect your path to be? Or is webpack expected to be installed globally?